opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/NoEncryptionServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package de.adorsys.opba.protocol.facade.services;
+
+import de.adorsys.opba.protocol.api.services.EncryptionService;
+import org.springframework.context.annotation.Profile;
+
+@Profile("no-enc")
```

Comment:
Profile for non-managed beans is useless

Reply:
ok